### DISCUSSION TITLE
Data List: $Parent

### DISCUSSION THREAD

**Discussion by thebedroomcoder** [leekelleher/umbraco-contentment#255] [Question] [answered]
I'm looking to use the data list to populate dropdown boxes.  I have the following

- Container
    - Node 1
    - Node 2
    - Settings Folder
          - Dropdown value 1 (category)
          - Dropdown value 2 (category)
- Container
    - Node 1
    - Node 2
    - Settings Folder
          - Dropdown value 1 (category)
          - Dropdown value 2 (category)
          
 The problem is I have a dropdown on the Node document type.  I want it to show the values of the settings folder in the same container.
 
`$parent//* [@nodeTypeAlias = 'category']`

**Comment by leekelleher** [comment, marked as answer]
@craigacronin Would this XPath work for you? `$current/../settingsFolder/category`

**Comment by thebedroomcoder** [comment]
I was about to abort but managed to make it work with `3]/configFolder/category`

Thanks Lee, the package saved me….Perfect solution to my problem.
